roles/prometheus-node-exporter/templates/docker.node-exporter.service.j2

```diff
@@ -9,7 +9,7 @@ Restart=always
 ExecStartPre=-/usr/bin/docker stop %n
 ExecStartPre=-/usr/bin/docker rm %n
 ExecStartPre=/usr/bin/docker pull {{ node_exporter_container }}
-ExecStart=/usr/bin/docker run --rm --network host --cpus="{{ node_exporter_max_cpu }}"  --pid="host" --name %n -v {{ node_exporter_prom_dir }}:/run/prometheus {{ node_exporter_container }} --collector.textfile.directory="{{ node_exporter_prom_dir }}"
+ExecStart=/usr/bin/docker run --rm --network host --cpus={{ node_exporter_max_cpu }}  --pid=host --name %n -v {{ node_exporter_prom_dir }}:/run/prometheus {{ node_exporter_container }} --collector.textfile.directory="{{ node_exporter_prom_dir }}"
```

Comment:
Similar, why drop the quotation marks?

Reply:
In my testing on RHEL8 for some reason systemd did not like quotes around those parameters. It would error out on that docker command complaining about the `--cpus` option and in another case about the `--pid` option. I finally got it working when I removed quotes. The docker commands themselves worked fine, just within systemd running these docker commands RHEL8 would choke on these particular quote parameters.

The values don't need quotes since there's not spaces or conventional delimiters in the parameters so it works fine without quotes on Ubuntu as well. But again on RHEL8 it refused to work. 

Again, this is not related to the PR per se but I had to fix this for a client's install and since I did it in the "single-node-slurm" branch I just committed these changes there.

Reply:
Got it, ok. Feel free to leave this in, and we can test later on RHEL8 to better understand the issue.